### DISCUSSION TITLE
Bots unequip their gear when I ditch them

### DISCUSSION THREAD

**Discussion by perka123** [trickerer/Trinity-Bots#363] [General]
So while I dont want these guys to always follow me around all the time, i naturally wanna tell them "hey go and chill out in dalaran, ill come get you when i wanna do some dungeons" and that makes them unequip all the stuff ive given them back into my bags

Any way to make them not? I tried telling them "wait here" instead but they randomly teleport to me when I'm too far away etc

**Comment by trickerer** [comment]
What you want is `.npcbot unbind` command, it removes bot temporarily, check the README on how it works
